app/resolve/daemon/__init__.py

```diff
@@ -5,12 +5,26 @@
 from core.parser import parser
 from core.rquest_dto.result import RquestResult
 from core.task_api_client import TaskApiClient
+from core.results_modifiers import result_modifiers
+import json
 
 
 def main() -> None:
     client = TaskApiClient()
     logger = logging.getLogger(settings.LOGGER_NAME)
-    result = execute_query(parser)
+    modifiers_list = result_modifiers(
+        low_number_suppession_threshold=int(
+            settings.LOW_NUMBER_SUPPRESSION_THRESHOLD or 0
+        ),
+        rounding_taget=int(settings.ROUNDING_TARGET or 0),
```

Comment:
```suggestion
        rounding_target=int(settings.ROUNDING_TARGET or 0),
```
this typo needs fixing in the `result_modifiers` function as well